Automattic/wp-calypso
# Sites dashboard preview - adopt primary wp admin button.

Related to # https://github.com/Automattic/dotcom-forge/issues/6829 

## Proposed Changes

cc @cleacos since we are updating the a4a component.  The scope of change seems limited to the dotcom implementation.

* Updates the position and style of the WP Admin button in GSV. While this is in the `item-preview-pane-header` that is shared between a4a and dotcom, the [WP admin link was recently added](https://github.com/Automattic/wp-calypso/pull/89894) as a conditional render from the dotcom side so this should have no effect on the a4a side.
* Updates the close button style and placement IFF the WP Admin button is present. Otherwise, the old style close button with the "X" icon is used. This should ensure no changes are visible on the a4a side WRT this button either.

BEFORE

<img width="1325" alt="Screenshot 2024-05-01 at 11 43 56 AM" src="https://github.com/Automattic/wp-calypso/assets/28742426/5c84f7cf-3712-4244-b1e5-38ee3d141654">


AFTER

<img width="1469" alt="Screenshot 2024-05-02 at 10 43 19 AM" src="https://github.com/Automattic/wp-calypso/assets/28742426/37619f0d-a8bb-4565-b73d-a5cc1e8fde79">



## Testing Instructions



* Test the calypso build and verify the wp admin link is now a button in the top right, and a secondary close button is positioned to the left of it.
* Test the a4a build and verify there are no changes.

## Pre-merge Checklist
